localstack/services/resourcegroups/provider.py

```diff
@@ -0,0 +1,7 @@
+from abc import ABC
+
+from localstack.aws.api.resource_groups import ResourceGroupsApi
+
+
+class ResourceGroupsProvider(ResourceGroupsApi, ABC):
```

Comment:
nit: i would remove the ABC here, because that would suggest we don't instantiate the class anywhere.